packages/fx-core/src/common/tools.ts

```diff
@@ -533,6 +533,6 @@ export function compileHandlebarsTemplateString(templateString: string, context:
 
 export function getArmOutput(ctx: PluginContext, key: string): string | undefined {
   const solutionConfig = ctx.configOfOtherPlugins.get("solution");
-  const output = solutionConfig?.get("armTemplate");
+  const output = solutionConfig?.get("armTemplateOutput");
```

Comment:
Please use the constant value `ARM_TEMPLATE_OUTPUT` in *packages\fx-core\src\plugins\solution\fx-solution\constants.ts*.